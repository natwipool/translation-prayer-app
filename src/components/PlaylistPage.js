import React from 'react';
import PlaylistLists from './PlaylistLists';

const PlaylistPage = (props) => (
  <div>
    <PlaylistLists />
  </div>
);

export default PlaylistPage;
