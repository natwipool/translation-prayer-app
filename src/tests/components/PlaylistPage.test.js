import React from 'react';
import { shallow } from 'enzyme';
import PlaylistPage from '../../components/PlaylistPage';

test('should render PlaylistPage correctly', () => {
  const wrapper = shallow(<PlaylistPage />);
  expect(wrapper).toMatchSnapshot();
});