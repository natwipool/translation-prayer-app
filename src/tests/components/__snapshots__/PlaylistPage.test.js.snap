// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PlaylistPage correctly 1`] = `
<div>
  <Connect(PlaylistLists) />
</div>
`;
